python/ray/util/placement_group.py

```diff
@@ -109,9 +109,7 @@ def bundle_count(self):
         return len(self.bundle_cache)
 
     def _get_none_zero_resource(self, bundle: List[Dict]):
-        # This number shouldn't be changed.
-        # When it is specified, node manager won't warn about infeasible
-        # tasks.
+        # Any number between 0-1 should be fine.
         INFEASIBLE_TASK_SUPPRESS_MAGIC_NUMBER = 0.0101
```

Comment:
Looks like this isn't used anymore after this change -- can you please remove it from here and `node_manager.cc` if that's the case?

Reply:
ah yeah LGTM. let's rename it to just magic number or something, the name is pretty confusing now that it doesn't actually suppress anything